src/store/page/page-element.ts

```diff
@@ -144,9 +152,27 @@ export class PageElement {
 			});
 		}
 
+		if (json.contents) {
+			const slots = json.contents as JsonObject;
+
+			Object.keys(slots).forEach(slotId => {
+				(slots[slotId] as JsonArray).map(
+					childElement =>
+						PageElement.fromJsonObject(
+							childElement as JsonObject,
+							element,
+							slotId
+						) as PageElement
+				);
+			});
+		}
+
+		// Migrate old children structure
 		if (json.children) {
-			element.children = (json.children as JsonArray).map(
```

Comment:
Now that we have the migration package, we should move this code.

Reply:
afaik we don't have a migration package just yet. we only talked about the plan to implement it that way. @LKuechler wanted to pull all in-code migration paths into separate packages as next steps.